railties/test/generators/shared_generator_tests.rb

```diff
@@ -51,7 +51,7 @@ def test_implied_options_with_conflicting_option
   end
 
   def test_skeleton_is_created
-    run_generator
+    run_generator [destination_root]
```

Comment:
```suggestion
    run_generator
```
This is probably not required.